plugins/SitesManager/Controller.php

```diff
@@ -228,7 +228,7 @@ public function siteWithoutDataTabs()
             'googleAnalyticsImporterMessage' => $googleAnalyticsImporterMessage,
             'tagManagerActive' => $tagManagerActive,
             'consentManagerName' => false,
-            'cloudflare' => $this->siteContentDetector->cloudflare,
+            'cloudflare' => $this->siteContentDetector->cloudflare
```

Comment:
keeping the trailing slash after the last array item would be preferred
```suggestion
            'cloudflare' => $this->siteContentDetector->cloudflare,
```